src/validator.js

```diff
@@ -0,0 +1,19 @@
+import Ajv from "ajv";
+
+const ajv = new Ajv();
+const schema = {
+  "properties": {
```

Comment:
I brought this up earlier. I thought you fixed it? You don't need to wrap your URL in an object. You can validate a string by itself. `ajv.validate(schema, "http://localhost:8000")`

Reply:
So I tried just passing in the string for the uri. However, it didn't work, so I'm just leaving it as is for now.